.trellis/scripts/common/io.py

```
@@ -0,0 +1,37 @@
+
+def write_json(path: Path, data: dict) -> bool:
+    """Write dict to JSON file with pretty formatting.
+
+    Returns True on success, False on error.
+    """
+    try:
+        path.write_text(
+            json.dumps(data, indent=2, ensure_ascii=False),
+            encoding="utf-8",
+        )
+        return True
```

Ruff (0.15.18)

[warning] Consider moving this statement to an `else` block

(TRY300)